app/components/search-result.js

```diff
@@ -42,10 +42,9 @@ export default Ember.Component.extend(Analytics, {
     hyperlink: Ember.computed('result', function() {
         let re = null;
         for (let i = 0; i < this.get('result.providers.length'); i++)
```

Comment:
Assign the providers outside of the loop (using property look-ups within a loop is expensive)
```js
const resultProviders = this.get('result.providers'); // May need to call .toArray() because ember
```

Not sure why this needs to be a loop since the last iteration will win. There's no if statement with break inside the loop. I realize you didn't write that part, but this is a good time to revisit it.
```js
const re = this.providerUrlRegex[resultProviders[resultProviders.length - 1] || this.providerUrlRegex.OSF;
```
⬆️  Should be the same result for `re`. If it does need to be in a loop for a valid reason, please add a comment explaining that reason so we can have it documented.